nservicebus/azure-storage-queues/configuration.md

```diff
@@ -79,4 +79,11 @@ This feature can be enabled by specifying `.UseAccountNamesInsteadOfConnectionSt
 
 snippet:AzureStorageQueueUseAccountNamesInsteadOfConnectionStrings
 
-NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
\ No newline at end of file
+NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
+
+## Hashing algorithms
+If the queue is longer than [63 characters](https://msdn.microsoft.com/en-us/library/azure/dd179349.aspx), the Azure Storage Queues Transport uses a hashing algorithm to rename it. The default algorithm is `MD5`. In order to use `SHA1` instead, use the following configuration:
```

Comment:
@SimonCropp The hash is added as part of the queue name at the end to match the length criterion, so there's a non-zero chance that your queue will be discoverable by it. I'll register an issue for this and link to the comment of yours.
@MarcinHoppe I'll update.